Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonFlowLayoutPanel.cs

```diff
@@ -0,0 +1,34 @@
+﻿#region BSD License
+/*
+ *
+ *  New BSD 3-Clause License (https://github.com/Krypton-Suite/Standard-Toolkit/blob/master/LICENSE)
+ *  Modifications by Peter Wagner(aka Wagnerp) & Simon Coghlan(aka Smurf-IV), et al. 2024 - 2024. All rights reserved.
+ *
+ */
+#endregion
+
+namespace Krypton.Toolkit
```

Comment:
There is an existing query in the backlog that was answered as to why this is not needed in the KSuite

Reply:
Removed